sdks/cpp/README.md

```diff
@@ -115,34 +115,49 @@ For more information, you can also read the [SDK Overview](../), check out [sdk.
 When running on Agones, the above functions should only fail under exceptional circumstances, so please 
 file a bug if it occurs.
 
-## Dynamic and Static Libraries
-
-In the [releases](https://github.com/googlecloudplatform/agones/releases) folder
-you will find two archives for download that contain both static and dynamic libraries for building your
-game server on Linux:
-
-- `argonsdk-$(VERSION)-dev-linux-arch_64.tar.gz`: This includes all the 
-headers as well as dynamic and static libraries that are needed for development and runtime.
-- `argonsdk-$(VERSION)-runtime-linux-arch_64.tar.gz`: This includes just the dynamic libraries that 
-are needed to run a binary compiled against the SDK and its dependencies.
-
-### Building the Libraries
+## Building From Source
+To build Agones C++ SDK from sources you need to set up your development environment and install following tools:
+- CMake (https://cmake.org/download/)
+- OpenSSL (https://github.com/openssl/openssl)
+- Git (https://git-scm.com/downloads)
+- gRPC (https://github.com/grpc/grpc)
 
-If you want to build the libraries from Agones source, 
-the `make` target `build-sdk-cpp` will compile both static and dynamic libraries for Debian/Linux
-for your usage, to be found in the `bin` directory inside this one.
+To build gRPC you may also need to install next dependencies:
```

Comment:
The previous PR's documentation covered this really nicely, and specifically went through each dependency:
https://github.com/GoogleCloudPlatform/agones/pull/421/files#diff-eeb2d8fcc3782771feb2798fb61b5979R135